DEPS.bzl

```diff
@@ -8990,6 +8990,16 @@ def go_deps():
             "https://storage.googleapis.com/cockroach-godeps/gomod/go.etcd.io/etcd/client/v2/io_etcd_go_etcd_client_v2-v2.305.0.zip",
         ],
     )
+    go_repository(
```

Comment:
Looks like we now depend on both `go.etcd.io/etcd/raft/v3` as well as `go.etcd.io/raft/v3`. I don't know if this is a huge deal, but getting rid of the extra dependency would be good.

Reply:
Could this be some fallout from #93190? I assume the intention of that PR was to remove the dependency on the former? cc: @pavelkalinnikov

Let me see if I can resolve this issue separately.